lib/jetstream/api/stream.ex

```diff
@@ -345,17 +345,23 @@ defmodule Jetstream.API.Stream do
   @doc """
   Paged list of known Streams including all their current information.
 
+  ## Options
+
+  * `:offset`
```

Comment:
It would be great that this option is used for pagination.

Question: what about `limit`?

Reply:
Good question. I gave this a try and it looks like the nats server API doesn't accept `limit`.

Reply:
Ok